common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -17,7 +17,7 @@
                     navigation-group--@{edition.id.toLowerCase}
                     js-editionlise-secondary-nav"
                     data-edition="@{edition.id.toLowerCase}"
-                    @if(edition.id.toLowerCase != "uk") { hidden }> @* Our default edition is UK *@
+                    @if(edition.id.toLowerCase != Edition(request).id.toLowerCase) { hidden }>
```

Comment:
I thought the new header was not editionalized at all server-side. Did I understand it wrongly?